spec/citylist-spec.js

```diff
@@ -23,8 +23,8 @@ describe('CityList', function() {
             for (var i = 0, n = keys.length; i < n; i++) {
                 key = keys[i];
                 city = cities[key];
-                expect(city.id).toBeDefined();
-                expect(city.label).toBeDefined();
```

Comment:
Much nicer. A little improvement on the error output:

> Expected undefined to be defined 'City berlin requires property label.'.

becomes a bit more readable if printed like this:

> Expected undefined to be defined "City 'berlin' requires property 'label'."

Reply:
As I think that it is not possible to change the outer quotes from single to double, I have used double quotes within the message. Otherwise the output would look like this:

> Expected undefined to be defined 'City 'berlin' requires property 'label'.'